volestipy/.ipynb_checkpoints/metabolic_net_pipeline-checkpoint.ipynb

```diff
@@ -0,0 +1,1987 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
this file is not relevant with this PR. I propose to keep it to your fork for now and not in the code base of volesti

Reply:
Yes, sure. I will add those on the .gitignore file